_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1056,7 +1056,17 @@ mfxStatus VAAPIVideoProcessing::Execute(mfxExecuteParams *pParams)
     output_region.width  = outInfo->CropW;
     m_pipelineParam[0].output_region = &output_region;
 
-    m_pipelineParam[0].output_background_color = 0xff000000; // black for ARGB
+    m_pipelineParam[0].output_background_color = pParams->iColorfillColor;
+
+    if ( pParams->bDisableColorfill ||
```

Comment:
This is not in this patch: where bDisableColorfill is being set? do we have some exposed mediasdk uAPI for that?